utils-testing-rx/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1 @@
+<manifest package="pm.gnosis.tests.utils" />
```

Comment:
Same here. This module doesn't necessarily have to be an android module. If we keep it as a `Java library` we can re-use it and we generate artefacts that are not android dependent.